openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-sharer.js

```diff
@@ -26,6 +26,16 @@ var Sharer = (function() {
 			, autoOpen: false
 			, resizable: false
 		});
+		
+		// Fix to move the close icon on top of the .ui-dialog-titlebar cause otherwise 
+		// touch-events are broken and you won't be able to close the dialog
+		var closeIcon = sharer.parent().find('.ui-dialog-titlebar-close').clone();
```

Comment:
I believe the easier and more straight-forward way would be:
`sharer.parent().find('.ui-dialog-titlebar-close').appendTo(sharer.parent())`

other code can be dropped ...

BTW I'm trying to use `let/const` instead of `var` in new code :)))

Reply:
Done. But I found some more dialogs. I created separated tickets for those, cause I can't find the UI code that triggers the dialog.

Reply:
Do the search using `.dialog({` string :) I got 7 occurrences